var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -103,7 +103,7 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
         "breakpad",
         default=True,
         description="Enable breakpad crash dump library",
-        when="@1.9:1.11",
+        when="@1.10:1.11",
```

Comment:
Seems like 1.9 can build with external breakpad, but 1.10–1.11 can only build with third party breakpad. And 1.9 has no way of controlling whether or not breakpad gets built. I think it's safer to ignore it on 1.9.